src/Functional/Functional.php

```diff
@@ -0,0 +1,439 @@
+<?php
+
+namespace Functional;
+
+final class Functional
+{
+    /**
+     * @see \Functional\average
+     */
+    const average = '\Functional\average';
```

Comment:
Btw value (_and all others as well_) can be defined like this: `const average = __NAMESPACE__ . '\average'`, which is more dynamic and explicit I guess..

Reply:
Happy to see a 2nd PR for that, merging the PR as is